app/assets/javascripts/application.js

```diff
@@ -12,6 +12,7 @@
 //= require theater
 //= require validate_file_input_fields
 //= require main
+//= require observation_form_map
```

Comment:
Would it be better to have app/views/observer/_form_observations.html.erb load this instead?  See how Ray had app/views/observer/_form_multi_image_upload.html.erb load the interactive image uploader.  What do you think?  I guess if it's really short it may be better to have it in the default JS slug that every page loads, than to have create/edit_observation have to load several additional files -- a few extra bytes in one download is quicker than a bunch of tiny downloads.  Not sure what's best.